content/markdown/installation.markdown

```diff
@@ -9,26 +9,26 @@
 
 To install PortMaster via a simple Installation Script download either the Install.PortMaster.sh or Install.Full.PortMaster.sh (with all Runtimes already included) from the link above and place it in the appropriate folder for your CFW. After that simply execute the .sh from your Ports folder.
 
-| CFW          | Location               |
-|--------------|------------------------|
-| AmberElec    | /roms/ports/           |
-| ArkOS        | /roms(or roms2)/ports/ |
-| JELOS        | /roms/ports/           |
-| UnofficialOS | /roms/ports/           |
+| CFW          | Location                      |
+|--------------|-------------------------------|
+| AmberElec    | /roms/ports/                  |
+| ArkOS        | /roms/tools/ or /roms2/tools/ |

```

Comment:
For the full install.portmaster.sh this is the wrong path. 
Here is still /roms/ports/ valid.